src/prefect/tests/quality/test_static.py

```diff
@@ -44,9 +48,13 @@ def test_run_api_db_validation_by_amenageur(monkeypatch):
     expected = 3
     assert len(report.results) == expected
 
-    # All amenageurs should pass tests
+    # (almost) All amenageurs should pass tests
     assert [r.amenageur for r in report.results] == ["Tesla", "Ionity", "Electra"]
-    for results in report.results:
-        assert results.success
-        successes = [s.success for s in results.suite]
+    for results in report.results:  # results: QCExpectationsSuiteResult
+        for result in results.suite:
+            if result.code != "NE10" and (
+                results.amenageur != "Tesla" or result.code != "PDLM"
+            ):
+                assert result.success
+        successes = [s.success for s in results.suite]  # s: QCExpectationResult
         assert len(successes) == len(results.suite)
```

Comment:
Instead of updating existing tests this way, it may be better to split tests by "amenageur" and distinguish what is expected and what is not. 

Our test database can fail in some tests: it's a good thing!